offer-manager-domain/src/test/groovy/life/qbic/portal/portlet/offers/OfferSpec.groovy

```diff
@@ -181,7 +184,7 @@ class OfferSpec extends Specification {
                 BigDecimal.valueOf(primaryAnalysisItem.product.externalUnitPrice)) * primaryAnalysisItem.quantity
         double expectedNetSum = (10.0 + (400 * 1.0) - totalDiscount)
         double expectedOverhead = (expectedNetSum) * 0.2
-        double expectedTaxes = (expectedNetSum + expectedOverhead) * 0.19
+        BigDecimal expectedTaxes = (expectedNetSum + expectedOverhead) * 0.19
```

Comment:
would you need to cast the double values also to BigDecimal to make the value exact?

Reply:
yes indeed 👍🏽